spec/backupsss/tar_spec.rb

```diff
@@ -93,5 +93,33 @@
 
       it { is_expected.to be_a(File) }
     end
+
+    context 'compress archieve' do
+      before do
+        allow(File).to receive(:writable?) { true }
+        allow(File).to receive(:readable?) { true }
+        allow(File).to receive(:open)      { true }
+        allow(Open3).to receive(:capture3).with('some/src', 'some/dest.tar')
+      end
+
+      subject { Backupsss::Tar.new('some/src', 'some/dest.tar') }
```

Comment:
If we go with my earlier comment about removing the attr_accessor for ``@compress_archive``, then we'll need an additional subject for the false case.